pkg/generate/templateset/vars.go

```diff
@@ -34,6 +31,9 @@ type MetaVars struct {
 	// for custom resources, e.g. "sns.services.k8s.aws" or
 	// "sfn.services.k8s.aws"
 	APIGroup string
+	// AWSSDKModelServiceID is the exact string that appears in the AWS service API's
+	// api-2.json descriptor file under `metadata.serviceId`
+	AWSSDKModelServiceID string
```

Comment:
this probably could have just been `SDKModelServiceID` to match the `SDKAPIInterfaceTypeName` field below.